src/test/java/LoginTests.java

```diff
@@ -23,118 +24,118 @@ public void setup() {
         driver.manage().deleteAllCookies();
     }
 
-    @Test
+    @Test(priority = 0)
```

Comment:
test deben ser independientes, no tiene sentido poner prioridad

Reply:
only for learning purpose. Deleted